Sources/HaCWebsiteLib/Workshop.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import Yaml
+import KituraMarkdown
+
+public struct Workshop {
+  let title: String
+  let authors: [String]
+  let presenters: [String]
+  let thanks: [Thanks]
+  let recommendations: [Recommendation]
+  let links: [URL]
+  let tags: [String]
+  let prerequisites: String
+  let description: String
+
+  struct Thanks: Equatable {
+    let to: String
+    let reason: String?
+  }
+
+  struct Recommendation: Equatable {
+    let title: String
+    let url: URL
+  }
+
+  enum WorkshopError : Error {
+    case malformedMetadata
+  }
+}
+
+// Satisfy equatability for Workshop.Recommendation
```

Comment:
I'd argue it's not clear that they are here for satisfying the Equatable protocol otherwise.